src/System.Net.NameResolution/tests/FunctionalTests/GetHostEntryTest.cs

```diff
@@ -17,37 +17,31 @@ public class GetHostEntryTest
         {
             IPAddress localIPAddress = await TestSettings.GetLocalIPAddress();
 
-            TestGetHostEntryAsync(() => Dns.GetHostEntryAsync(localIPAddress));
+            await TestGetHostEntryAsync(() => Dns.GetHostEntryAsync(localIPAddress));
         }
 
         [Theory]
         [InlineData("")]
         [InlineData(TestSettings.LocalHost)]
-        public void Dns_GetHostEntry_HostString_Ok(string hostName)
-        {
-            TestGetHostEntryAsync(() => Task.FromResult(Dns.GetHostEntry(hostName)));
-        }
+        public Task Dns_GetHostEntry_HostString_Ok(string hostName) 
+            => TestGetHostEntryAsync(() => Task.FromResult(Dns.GetHostEntry(hostName)));
```

Comment:
Nit: we generally put the `=>` on the same line as the signature.

Reply:
Done